resources/completions/BetterJinja (Filters).sublime-completions

```diff
@@ -1,5 +1,5 @@
 {
-    "scope": "text.jinja text.html.basic & (meta.placeholder.jinja | meta.statement.jinja)",
+    "scope": "(text.jinja, text.html.jinja, source.yaml.jinja) & (meta.interpolation.jinja | meta.statement.jinja)",
```

Comment:
I wonder if defining a meta scope `meta.jinja` and using that in all Jinja-enabled syntaxes would simplify this a lot, especially if more are were going to be added.

Reply:
Sounds useful. Actually something we probably should consider for bundled syntaxes as well.

Snippets/Completions selectors is a common issue, which I haven't paid much attention before.